addon/components/labs-ui/legend-icon.hbs

```diff
@@ -0,0 +1,19 @@
+<span class="legend-icon">
+<span class={{this.iconType}}>
+  {{#each @icon.layers as |icon-layer|}}
+
+    {{#if (eq @icon.type "line")}}
```

Comment:
If I understand correctly, `@iconType` would just use the argument exactly as it is passed. I've used `this.iconType` here because it's computed based on what's passed. 

(Of course, the computed could be refactored so it's unnecessary and we just use the passed arg. It just changes the passed string 'fa-icon' to 'fa-layers'. But I just ported this functionality as it was in the old version because I didn't want to mess with anything that involves labs-layers-api.)

Reply:
I've noted this with an inline code comment and will create a separate issue.